Doc/c-api/gcsupport.rst

```diff
@@ -65,6 +65,20 @@ rules:
    Analogous to :c:func:`PyObject_NewVar` but for container objects with the
    :const:`Py_TPFLAGS_HAVE_GC` flag set.
 
+.. c:function:: TYPE* PyUnstable_Object_GC_NewWithExtraData(PyTypeObject *type, size_t extra_size)
+
+   Analogous to :c:func:`PyObject_GC_New` but for container objects that
+   have additional data at the end of the object not managed by Python.
+
+   .. warning::
+       The function is marked as unstable because the final mechanism
+       for reserving extra data after an instance is not yet decided.
+       Once `PEP-697<https://peps.python.org/pep-0697/>`_ is
```

Comment:
```suggestion
       Once :pep:`697` is
```

Reply:
Thanks, I was wondering how that's done :+1:  Fixed